experimenter/experimenter/nimbus-ui/src/components/Summary/SummaryTimeline/index.test.tsx

```diff
@@ -81,6 +85,32 @@ describe("SummaryTimeline", () => {
     ).toHaveAttribute("data-tip", TOOLTIP_DURATION);
   });
 
+  it("renders with a live first run experiment", async () => {
+    render(
+      <Subject
+        status={NimbusExperimentStatusEnum.LIVE}
+        isFirstRun={true}
+        proposedReleaseDate="2023-12-12"
+      />,
+    );
+
+    expect(innerBar().classList).toContain("progress-bar-animated");
+    expect(innerBar().classList).toContain("progress-bar-striped");
+
+    expect(screen.queryByTestId("label-not-launched")).not.toBeInTheDocument();
+    expect(screen.queryByTestId("label-start-date")).toBeInTheDocument();
```

Comment:
Replace with `screen.getByTestId(...);` which throws if it is not found.

Reply:
I'll change the ones I added and file a bug for changing the rest of them :D